K2Field.K2NE.ServiceBroker/Constants/Methods.cs

```diff
@@ -162,5 +162,10 @@ public static class ExcelDocumentServices
             public const string SaveMultipleCellValuesList = "SaveMultipleCellValuesList";
             public const string SaveMultipleCellValues = "SaveMultipleCellValues";
         }
+
+        public static class ExcelImportServices
+        {
+            public const string UploadExcelDataToASmartObject = "UploadExcelDataToASmartObject";
```

Comment:
Try not to use the work SmartObject in a broker, because a broker is not SmartObject aware. A SmartObject consists of multiple ServiceObjects.

I would shorten it to:
UploadData